### ISSUE TITLE
Provide defaults for environment variables

### ISSUE THREAD

**Issue by maxnoe** [jitsi/docker-jitsi-meet#210]
I was trying to setup the jitsi stack in rancher, which does not support uploading a env file.
So I had to put in every environment variable into the docker file itself.

It would be nice, if the "defaults you probably won't have to change" would be actual defaults, so they do not need to be set.

**Comment by MaxNoe** [comment]
For the moment I wrote a small python script to fill in the values:

```python
import re
from dotenv import dotenv_values

with open('./docker-compose.yml') as f:
    content = f.read()


env = dotenv_values('.env')

for var, value in env.items():
    content = re.sub(rf'- {var}', f'- {var}={value}', content)
    content = re.sub(rf'\${{{var}}}', value, content)

with open('docker-compose-with-env.yml', 'w') as f:
    f.write(content)```

**Comment by saghul** [comment]
Makes sense, we could also remve them from the env file in that case. Are you up for writing a PR?

**Comment by MaxNoe** [comment]
yep

**Comment by sapkra** [comment]
If we have the env vars including default values in the Dockerfiles we should also remove the default values from the config files.

For example:
https://github.com/jitsi/docker-jitsi-meet/blob/ffa017b74d57ba2580fe6647f8de06b012aa4ef2/prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua#L5

**Comment by saghul** [comment]
@sapkra Agreed.
